lib/msal-common/src/index.ts

```diff
@@ -140,6 +140,7 @@ export { NativeRequest } from "./request/NativeRequest";
 export { NativeSignOutRequest } from "./request/NativeSignOutRequest";
 export { RequestParameterBuilder } from "./request/RequestParameterBuilder";
 export { StoreInCache } from "./request/StoreInCache";
+export { RequestValidator } from "./request/RequestValidator";
```

Comment:
Why was this added?

Reply:
We're using it in msal-node now to validate claims provided by a developer